### DISCUSSION TITLE
not recommended not working

### DISCUSSION THREAD

**Discussion by louwonder** [AprilSylph/XKit-Rewritten#1342] [Help Desk]
hey- I'm guessing I'm not the only one suddenly getting the full array of recommended blogs interrupting my dash? I assume they made a change on tumblr end and you fine folks are working on it but it's been a few days so in case it's just me, checking in

**Comment by marcustyphoon** [comment]
Yep, see #1335!

**Reply by louwonder** [reply]
ah yes I see sorry I missed it in there. Hope it can be sorted soon and thank for everything!
